_episodes/09-pull-requests.md

```diff
@@ -160,23 +238,40 @@ To https://github.com/USERNAME/countries.git
 {: .output}
 
 Next go to your forked github version of the repo and reload the page.
-You should see the changes in your version of the repo.
+You won't see the new file added in the list of files but you will see 
+that you recently pushed a new branch to the repository.
+
+![](../fig/github_screenshot_origin_master_addFrance.png)
+
+If you wish to view your new branch you can click on the 'Branch' drop down menu
+and select that branch.
+
+![](../fig/github_screenshot_switch_github_branch.png)
 
-![](../fig/github_screenshot_origin_wFrance.png)
+Then you should be able to view the files and commits in that branch.
+
+![](../fig/github_screenshot_origin_master_addFrance.png)
+
+Github already suspects that we are going to want to make a pull request so we can click
+the 'Compare & pull request' button to start a new pull request.
 
 Then click the 'New pull request' button.
 
 ![](../fig/github_screenshot_makingPR1.png)
 
-The base fork should be the upstream version and then the head fork should be your version.
-Then click the 'create pull request button'.
+The base fork should be the upstream/authoritative version's master branch and then 
+the head fork should be the new branch of our fork.
+You can add more information into the comment section if there is anything you'd like 
+to add for the person who reviews your suggestion.
+Then you can click the 'create pull request button' to submit the pull request.
 
 ![](../fig/github_screenshot_makingPR2.png)
 
 Now someone with privileges to the upstream repo can review it, give comments and
 suggestions, and merge it into the upstream version.
 
-Let's add a line to our file and see how/if this changes our pull request.
+Our collaborator reviewing the pull request noticed that 
```

Comment:
I think having another instructor leave a comment is a great idea!  Not sure if it is needed to show them the other instructor's screen (unless we do inline comments).
I will try to add some sentences about what you can see in a PR.